modules/cluster-logging-exported-fields-top-level-fields.adoc

```diff
@@ -0,0 +1,167 @@
+[id="cluster-logging-exported-fields-top-level-fields_{context}"]
+== Top-level fields
+
+// Normally, the preceding title would be an H1 prefixed with an `=`. However, because the following content is auto-generated at https://github.com/ViaQ/documentation/blob/main/src/data_model/public/top-level.part.adoc and pasted here, it is more efficient to use it as-is with no modifications. Therefore, to "realign" the content, I am going to prefix the title with `==` and use `include::modules/cluster-logging-exported-fields-top-level-fields.adoc[leveloffset=0]` in the assembly file.
+
+// DO NOT MODIFY THE FOLLOWING CONTENT. Instead, update https://github.com/ViaQ/documentation/blob/main/src/data_model/model/top-level.yaml and run `make` as instructed here: https://github.com/ViaQ/documentation
+
+//The top-level fields can be present in every record. The descriptions for fields that are optional begin with "Optional."
```

Comment:
@aconway Weird phrase. Maybe we should have an introductory phrase like this one. WDYT?